theano/compile/function_module.py

```diff
@@ -1428,6 +1429,8 @@ def create(self, input_storage=None, trustme=False):
         if self.profile:
             self.profile.linker_time += linker_time
             _fn.time_thunks = self.profile.flag_time_thunks
+            import_time = theano.gof.cmodule.import_time - start_import_time
```

Comment:
I'm not sure how that would be non-zero, since `start_import_time` has been defined as `theano.gof.cmodule.import_time`.